src/components/dashboard/Dashboard.jsx

```diff
@@ -156,6 +173,58 @@ export default function Dashboard() {
             )}
           </div>
         </DashboardPanel>
+        <DashboardDrawer open={!(_isEmpty(activeCourse))} onClose={onDraweClose}>
+          { activeCourse && (
+            <CourseDetails
+              title={activeCourse.title}
+              description={activeCourse.full_description}
+              details={[
+                {
+                  key: 'Time investment',
+                  value: `${activeCourse.hours_required} hours`,
+                  icon: <Alarm />,
+                },
+                {
+                  key: 'Certificate',
+                  value: activeCourse.has_certificate ? 'Avaliable' : 'Not avaliable',
+                  icon: <Certificate />,
+                },
+                {
+                  key: 'Difficulty level',
+                  value: activeCourse.difficulty_level,
+                  icon: <Dash />,
+                },
+                {
+                  key: 'Primary language',
+                  value: activeCourse.primary_language,
+                  icon: <World />,
+                },
+                {
+                  key: 'Subtitles',
+                  value: activeCourse.subtitles_available ? 'Available' : 'Not avaliable',
+                  icon: <Baseline />,
+                },
+                {
+                  key: 'Prerequisites',
+                  value: activeCourse.prerequisites,
+                  icon: <Checklist />,
+                },
+              ]}
```

Comment:
We can use something like: 

```suggestion
              ].filter(item => !!item.value)}
```

to prevent displaying of items with no value returned from BE.